.circleci/config.yml

```diff
@@ -34,8 +34,7 @@ commands:
 jobs:
   install-operator:
     docker:
-      - image: circleci/golang:1.17.3
-    working_directory: /go/src/github.com/3scale/apicast-operator
```

Comment:
was this meant to be removed ?

Reply:
I see its done here also https://github.com/3scale/apicast-operator/pull/204/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L154 so guess its intentional